local/manifest-pipeline-deploy.sh

```diff
@@ -0,0 +1,39 @@
+PROGNAME=$0
```

Comment:
Jon and I talked, he's going to move this into the other project to use when deploying locally during app code iteration. Just marking it here to keep track of this